src/flex-example-app-ios.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<application xmlns="http://ns.adobe.com/air/application/19.0">
+    <id>com.EnterpriseHeyzap.HeyzapSDKTest</id>
+    <filename>flex-example-app</filename>
+    <name>flex-example-app</name>
+    <versionNumber>1.0.0</versionNumber>
+    <extensions>
+        <extensionID>com.heyzap.sdk.extensions</extensionID>
+
+        <extensionID>com.heyzap.sdk.extensions.facebook.audiencenetwork</extensionID>
+
+<!--
+        <extensionID>com.heyzap.sdk.extensions.admob</extensionID>
+        <extensionID>com.heyzap.sdk.extensions.unityads</extensionID>
+
+        <extensionID>com.jirbo.airadc.AirAdColony</extensionID>
+        <extensionID>com.chartboost.plugin.air</extensionID>
+
+        <extensionID>com.applovin.sdk.air</extensionID>
+        <extensionID>com.vungle.extensions.Vungle</extensionID>
+-->
+    </extensions>
+
+    <initialWindow>
+        <content></content>
+        <aspectRatio>portrait</aspectRatio>
+        <visible>true</visible>
+        <autoOrients>true</autoOrients>
+        <fullScreen>true</fullScreen>
+        <renderMode>direct</renderMode>
+    </initialWindow>
+
+    <iPhone>
+        <InfoAdditions>
+            <![CDATA[
+            <key>UIDeviceFamily</key>
+            <array>
+                <string>1</string>
+                <string>2</string>
+            </array>
+            <!--<key>UIStatusBarStyle</key>-->
+            <!--<string>UIStatusBarStyleBlackOpaque</string>-->
+            <key>MinimumOSVersion</key>
+            <string>8.0</string>
```

Comment:
App Lovin ANE has 8.0 as min req. Don't know why but actually about 86% of all iDevices have 8.0 and greater version according to official stats. So i think it's okay.